gif-screencast.el

```diff
@@ -102,6 +115,17 @@ various programs run here."
   :group 'gif-screencast
   :type 'directory)
 
+(defcustom gif-screencast-capture-format "png"
+  "Image format to store the captured images.
+If you are an OSX user, \"ppm\" should be specified."
```

Comment:
I believe OSX is now called `macOS`.

Reply:
Right!! Reflected to all existing `OSX` in the code.